app/src/main/java/com/github/dedis/student20_pop/home/HomeViewModel.java

```diff
@@ -151,6 +162,24 @@ public void handleResult(Result result) {
 
     }
 
+    public void launchNewLao(String laoName) {
```

Comment:
I think the argument can be removed and the name can be retrieved from `mLaoName`